### PR TITLE
[HOPSWORKS-1617] Resolve host from IP using ohai's fqdn

### PR DESCRIPTION
**Pull request by jimdowling**
https://logicalclocks.atlassian.net/browse/HOPSWORKS-1617

### CODE REVIEW

File: recipes/default.rb [hunk 1]
```diff
@@ -191,17 +191,20 @@
 
 for h in node['kagent']['default']['private_ips']
 
-  # Try and resolve hostname first using /etc/hosts, then use DNS
-  begin
-    hname = hostf.getname(h)
-  rescue
+  if node['install']['localhost'].eql? "true"
```
Comment (kouzant) [inline recipes/default.rb]: Why's that?

File: Berksfile [hunk 2]
```diff
@@ -4,8 +4,8 @@ source 'https://supermarket.chef.io'
 metadata
 
 cookbook 'conda', github: "logicalclocks/conda-chef", branch: "master"
-cookbook 'kagent', github: "logicalclocks/kagent-chef", branch: "master"
-cookbook 'hops', github: "logicalclocks/hops-hadoop-chef", branch: "master"
+cookbook 'kagent', github: "logicalclocks/kagent-chef", branch: "karamel_installer"
```
Comment (SirOibaf) [inline Berksfile]: Remember to fix these before merging

File: Berksfile [hunk 3]
```diff
@@ -4,8 +4,8 @@ source 'https://supermarket.chef.io'
...
+cookbook 'hops', github: "logicalclocks/hops-hadoop-chef", branch: "karamel_installer"
```
Comment (kouzant) [inline Berksfile]: Kind reminder to set back to `master` ;)

Comment (jimdowling) [reply Berksfile]: ok

**Pull request merged**